apps/client/src/common/components/view-params-editor/ParamInput.tsx

```diff
@@ -11,14 +11,19 @@ interface EditFormInputProps {
 
 export default function ParamInput({ paramField }: EditFormInputProps) {
   const [searchParams] = useSearchParams();
-  const { id, type } = paramField;
+  const { id, type, defaultValue } = paramField;
```

Comment:
took me while to figure out where `paramField` comes from.
According to git blame, it was actually me writing this ... can you please correct my mistake and name this a prop? 😅

Reply:
Resolved with https://github.com/cpvalente/ontime/pull/610/commits/41ec894071e1490b26664daa511d7afa97c6fb51